com.unity.robotics.urdf-importer/Runtime/UnityMeshImporter/Plugins/AssimpNet/AssimpNet.xml

```diff
@@ -2209,7 +2209,7 @@
         </member>
         <member name="T:Assimp.Configs.IRRAnimationFrameRateConfig">
             <summary>
-            Defines the output frame rate of the IRR loader.
+            Defines the Output frame rate of the IRR loader.
```

Comment:
revert this file. This looks like an unintended consequence of doing a mass find-replace.